generated/nidcpower/nidcpower_service.cpp

```diff
@@ -2168,7 +2168,30 @@ namespace nidcpower_grpc {
       return ::grpc::Status::CANCELLED;
     }
     try {
-      return ::grpc::Status(::grpc::UNIMPLEMENTED, "TODO: This server handler has not been implemented.");
+      auto vi_grpc_session = request->vi();
+      ViSession vi = session_repository_->access_session(vi_grpc_session.id(), vi_grpc_session.name());
+      ViConstString channel_name = request->channel_name().c_str();
+      ViReal64 timeout = request->timeout();
+      ViInt32 count = request->count();
+      response->mutable_voltage_measurements()->Resize(count, 0);
+      ViReal64* voltage_measurements = response->mutable_voltage_measurements()->mutable_data();
+      response->mutable_current_measurements()->Resize(count, 0);
+      ViReal64* current_measurements = response->mutable_current_measurements()->mutable_data();
+      std::vector<ViBoolean> in_compliance(count, ViBoolean());
+      response->mutable_in_compliance()->Resize(count, 0);
+      auto in_compliance_mutable_data = response->mutable_in_compliance()->mutable_data();
+      ViInt32 actual_count {};
+      auto status = library_->FetchMultiple(vi, channel_name, timeout, count, voltage_measurements, current_measurements, (ViBoolean*)in_compliance.data(), &actual_count);
+      response->set_status(status);
+      if (status == 0) {
+        int i = 0;
+        for (auto item : in_compliance) {
+            in_compliance_mutable_data[i] = item;
```

Comment:
The spacing should have been 2 spaces less I guess. You can update mako file to adjust spacing in auto generated file